pages/services/prediction-agents.jsx

```diff
@@ -96,16 +94,16 @@ const PredictionAgents = () => (
       }}
       body={{
         steps: [
+          // eslint-disable-next-line react/jsx-key
           <ExternalLink href="https://github.com/valory-xyz/trader-quickstart?tab=readme-ov-file#system-requirements">
             Get the requirements in place
           </ExternalLink>,
+          // eslint-disable-next-line react/jsx-key
```

Comment:
we discussed that with @atepem, I don't think it makes sense to add a key here because these steps are mapped and have their on keys in 'li' tags in render, here. so it just eslint complaining, @mohandast52 do you think we should better have keys here anyway?
![image](https://github.com/user-attachments/assets/189c66a7-3fd8-44a2-b24b-7c30d54b1fb8)

Reply:
Ohh ok sounds good, I'll add these changes in with the next website-updates PR :)